File: Linguagem C Exercicios/C-Condicional/LANCHONETE.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    int codigo,qnt = 0;
    double valorTotal = 0;

    printf("Digite o codigo do produto: ");
    scanf("%d",&codigo);
    printf("Digite a quantidade que deseja comprar: ");
    scanf("%d",&qnt);

    if(codigo == 1){
            valorTotal = qnt * 5.00;
    }else if(codigo == 2){
        valorTotal = qnt * 3.50;
    }else if(codigo == 3){
        valorTotal = qnt * 4.80;
    }else if(codigo == 4){
        valorTotal = qnt * 8.90;
    }else if(codigo == 5){
        valorTotal = qnt * 7.32;

    }else{
      printf("Código invalido !!");
    };

    printf("Valor total a pagar: R$ %.2lf",valorTotal);


    return 0;
}

File: Linguagem C#/Condicional/Lanchonete.cs
using System;
using System.Globalization;

namespace Lanchonete {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;

            double valorTotal;
            int qnt, codigo;

            Console.Write("Codigo do produto comprado: ");
            codigo = int.Parse(Console.ReadLine());

            switch (codigo) {
                case 1:
                    Console.Write("Digite a quantidade desejada: ");
                    qnt = int.Parse(Console.ReadLine());
                    valorTotal = qnt * 5.00;
                    Console.WriteLine("Valor a pagar: "+valorTotal.ToString("F2",CI));
                    break;
                case 2:
                    Console.Write("Digite a quantidade desejada: ");
                    qnt = int.Parse(Console.ReadLine());
                    valorTotal = qnt * 3.50;
                    Console.WriteLine("Valor a pagar: " + valorTotal.ToString("F2", CI));
                    break;
                case 3:
                    Console.Write("Digite a quantidade desejada: ");
                    qnt = int.Parse(Console.ReadLine());
                    valorTotal = qnt * 4.80;
                    Console.WriteLine("Valor a pagar: " + valorTotal.ToString("F2", CI));
                    break;
                case 4:
                    Console.Write("Digite a quantidade desejada: ");
                    qnt = int.Parse(Console.ReadLine());
                    valorTotal = qnt * 8.90;
                    Console.WriteLine("Valor a pagar: " + valorTotal.ToString("F2", CI));
                    break;
                case 5:
                    Console.Write("Digite a quantidade desejada: ");
                    qnt = int.Parse(Console.ReadLine());
                    valorTotal = qnt * 7.32;
                    Console.WriteLine("Valor a pagar: " + valorTotal.ToString("F2", CI));
                    break;
                default:
                    Console.WriteLine("Código invalido !!");
                    break;
            };
        }
    }
}
